lisa/variable.py

```diff
@@ -10,6 +10,7 @@
 import yaml
 
 from lisa import schema, secret
+from lisa.sut_orchestrator.azure.common import PATTERN_SASURL
```

Comment:
This change doesn't need to change the variable.py. It's a lower level lib, shouldn't import azure libs. The secrets should be marked when it's first used in azure or other filles. If it's hard to know, mark it on first imported by special runbook variable, env variable, or command line variable. All of them support to mark a value to secret. The runbook supports format, and it's not must have.

Reply:
I posted a new update as we discussed. Please help to review again.